celesta-sql/src/main/javacc/CelestaParser.jj

```diff
@@ -449,7 +449,6 @@ void references(ForeignKey fk):{
 
   "(" t = <S_IDENTIFIER> {fk.addReferencedColumn(t.toString());}
   ("," t = <S_IDENTIFIER> {fk.addReferencedColumn(t.toString());})* ")"
-{fk.finalizeReference();}
```

Comment:
You removed this because now all the reference resolving machinery work after parsing, right?

Reply:
Yes